ddl/table.go

```diff
@@ -490,6 +506,25 @@ func onShardRowID(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	return ver, nil
 }
 
+func checkGlobalAutoID(s *sessionPool, tbl table.Table, shardRowIDBits uint64) error {
```

Comment:
checkGlobalAutoID is not a good name. You need to read the code to know what it really does. what about `verifyNoOverflowShardBits` ?